docs/rtk-query/usage/examples.mdx

```diff
@@ -32,7 +32,7 @@ In the example below you'll notice a few things. There are two `Posts` list on t
 The example has some intentionally wonky behavior... when editing the name of a post, there is a decent chance you'll get a random error.
 :::
 
-<iframe src="https://codesandbox.io/embed/concepts-optimistic-updates-lush8?fontsize=12&hidenavigation=1&module=%2Fsrc%2Fapp%2Fservices%2Fposts.ts&theme=dark&runonclick=1"
+<iframe src="https://githubbox.com/reduxjs/redux-toolkit/tree/feature/v1.6-integration/examples/query/react/optimistic-update?fontsize=12&hidenavigation=1&module=%2Fsrc%2Fapp%2Fservices%2Fposts.ts&theme=dark&runonclick=1"
```

Comment:
```suggestion
<iframe src="https://codesandbox.io/embed/github/reduxjs/redux-toolkit/tree/feature/v1.6-integration/examples/query/react/optimistic-update?fontsize=12&hidenavigation=1&module=%2Fsrc%2Fapp%2Fservices%2Fposts.ts&theme=dark&runonclick=1"
```

I don't see a problem with this, but long term we should probably just use the built-in CSB feature.

Reply:
I thought so too, but apparently githubbox **is** a codesandbox feature. Just add `box` to any github link and you have a codesandbox

Reply:
I just realized that apparently https://githubbox.com cannot be embedded.